plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/PrefPageUsageStatistics.java

```diff
@@ -53,7 +54,7 @@ public void init(IWorkbench workbench) {
     protected Control createPreferenceContent(@NotNull Composite parent) {
         Composite composite = UIUtils.createPlaceholder(parent, 1);
 
-        Composite group = UIUtils.createControlGroup(composite, "Data sharing", 1, GridData.FILL_HORIZONTAL, SWT.DEFAULT);
+        Composite group = UIUtils.createControlGroup(composite, "Data share", 1, GridData.FILL_HORIZONTAL, SWT.DEFAULT);
```

Comment:
Why this change?

Reply:
In my opinion, it's not a suitable term. The term "data sharing" refers to the concept as a whole. On the other hand, the term "data share" refers to a particular sample of shared data.

Reply:
We discussed that this change is redundant. Please revert it.

Reply:
We don't have to change this heading in Preferences. I'm not sure that this is so important, but here Data sharing looks good.